fog/ledger/enclave/impl/src/lib.rs

```diff
@@ -8,29 +8,51 @@
 //! with all the unit testing they would ever need. Fate, it seems, has a sense
 //! of irony...
 
+#![allow(unused)]
 #![no_std]
-
 extern crate alloc;
 
+mod key_image_store;
 use alloc::vec::Vec;
-use fog_ledger_enclave_api::{KeyImageContext, LedgerEnclave, OutputContext, Result};
+use fog_ledger_enclave_api::{
+    messages::KeyImageData, AddRecordsError, Error, LedgerEnclave, OutputContext, Result,
+    UntrustedKeyImageQueryResponse,
+};
 use fog_types::ledger::{
     CheckKeyImagesRequest, CheckKeyImagesResponse, GetOutputsRequest, GetOutputsResponse,
 };
+use key_image_store::{KeyImageStore, StorageDataSize, StorageMetaSize};
 use mc_attest_core::{IasNonce, Quote, QuoteNonce, Report, TargetInfo, VerificationReport};
 use mc_attest_enclave_api::{ClientAuthRequest, ClientAuthResponse, ClientSession, EnclaveMessage};
-use mc_common::ResponderId;
+use mc_common::{
+    logger::{create_root_logger, log, Logger},
```

Comment:
This will not work I think, create_root_logger uses the rust stdlib, so this will break the build

Reply:
done removed create_root_logger